sections/email-signup-banner.liquid

```diff
@@ -157,23 +178,67 @@
     },
     {
       "type": "select",
-      "id": "desktop_text_box_position",
+      "id": "desktop_content_position",
+      "options": [
+        {
+          "value": "flex-start flex-start",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__1.label"
+        },
+        {
+          "value": "flex-start center",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__2.label"
+        },
+        {
+          "value": "flex-start flex-end",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__3.label"
+        },
+        {
+          "value": "center flex-start",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__4.label"
+        },
+        {
+          "value": "center center",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__5.label"
+        },
+        {
+          "value": "center flex-end",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__6.label"
+        },
+        {
+          "value": "flex-end flex-start",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__7.label"
+        },
+        {
+          "value": "flex-end center",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__8.label"
+        },
+        {
+          "value": "flex-end flex-end",
+          "label": "t:sections.email-signup-banner.settings.desktop_content_position.options__9.label"
+        }
+      ],
+      "default": "flex-end center",
+      "label": "t:sections.email-signup-banner.settings.desktop_content_position.label"
+    },
```

Comment:
I believe this is happening because of this: https://screenshot.click/24-49-rm2py-j9grp.mp4

Which is causing this: https://screenshot.click/24-50-qk557-lyrag.mp4

which causes us to have a smaller box when we have multiple sections 🤔 : https://screenshot.click/24-51-bawmk-hqqfu.mp4

I believe we can remove it, unless it's being used for something else I am missing 🤷‍♀️ 

We can have `newsletter__wrapper` always be 100% when the box is present

Reply:
It looks like this was added to ensure that when newsletter is the only section its full width:

https://screenshot.click/25-44-e3yq2-85f5d.mp4

Fixed here: https://github.com/Shopify/dawn/pull/900/commits/8ca0de6e7d74e52cba19a5b316dae33de9a077ee